egs/iam/v1/local/unk_arc_post_to_transcription.py

```diff
@@ -46,11 +47,11 @@
 word_handle = open(args.words, 'r', encoding='latin-1')
 unk_handle = open(args.unk,'r', encoding='latin-1')
 if args.one_best_arc_post == '-':
-    arc_post_handle = sys.stdin
+    arc_post_handle = io.TextIOWrapper(sys.stdin.buffer, encoding='latin-1')
```

Comment:
I am thinking of converting these latin-1 things to utf-8, since it seems to be more painful than I had thought to emulate python2's encoding-agnostic behavior in python3, and I think these days it's safe to assume no-one is using any other encoding.  Can you see if this works with utf-8 as the encoding type?

Reply:
I tried using utf-8 encoding, somehow it is degrading the WER. I am trying to find the reason.